woudc_data_registry/product/totalozone/totalozone_generator.py

```diff
@@ -76,8 +76,12 @@ def execute(path, bypass):
     registry_.save()
 
     # traverse directory of files
+    previously_seen_instrument = []
     for dataset in datasets:
         for dirname, dirnames, filenames in os.walk(dataset):
```

Comment:
```suggestion
    for dataset_path in datasets:
        for dirname, dirnames, filenames in os.walk(dataset_path):
```

Can you rename `dataset` to `dataset_path` here? I did this in other product generator scripts but I missed fixing this one. I wanted to avoid using just the name `dataset` because it was a bit too vague.
Please rename any `dataset` to `dataset_path` in the remaining `for` loop.; note to not mixup with `dataset_id` in certain cases because `dataset` was the old name for `dataset_id` too.